module_utils/network_lsr/argument_validator.py

```diff
@@ -2572,61 +2572,34 @@ class IPRouteUtils(object):
 
     @classmethod
     def _parse_route_tables_mapping(cls, file_content, mapping):
+        rr = re.compile(
+            b"^\\s*(0x[0-9a-fA-F]+|[0-9]+)\\s+([a-zA-Z0-9_.-]+)(\\s*|\\s+#.*)$"
```

Comment:
so a line like
```
0xffff somevalue# this is a comment
```
would not match - is this line allowed by iproute2?

Reply:
yes, it would not match. You can test by adding
```
44  ttt1# comment
45  ttt2 # comment
```
to `/etc/iproute2/rt_tables` and type `ip route add 1.2.3.4/32 dev eth0 table ttt1` (while `ttt2` works).